src/opentimelineio/clip.h

```diff
@@ -45,17 +45,17 @@ class Clip : public Item
                std::string const& new_active_key,
                ErrorStatus*       error_status = nullptr) noexcept;
 
-    virtual TimeRange
-    available_range(ErrorStatus* error_status = nullptr) const;
+    TimeRange
+    available_range(ErrorStatus* error_status = nullptr) const override;
 
-    virtual optional<Imath::Box2d>
-    available_image_bounds(ErrorStatus* error_status) const;
+    optional<Imath::Box2d>
+    available_image_bounds(ErrorStatus* error_status) const override;
 
 protected:
-    virtual ~Clip();
```

Comment:
My reading of the iso cpp guidelines is that destructors must be virtual, not override.

http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c127-a-class-with-a-virtual-function-should-have-a-virtual-or-protected-destructor

http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rh-override

Do you read it the same as I?